### PR TITLE
Migration: Create scripts for migration actions

### PR DESCRIPTION
**Pull request by danoctavian**
## Context

Issue: https://github.com/NexusMutual/smart-contracts/issues/707

## Changes proposed in this pull request

Add script for running commands that is also tested as part of the migration-v2.js script

## Test plan

Please describe the tests cases that you ran to verify your changes. Add further instructions on 
how to run them if needed (i.e. migration / deployment scripts, env vars, etc).

## Checklist

- [x] Rebased the base branch
- [x] Attached corresponding Github issue
- [ ] Prefixed the name with the type of change (i.e. feat, chore, test)
- [x] Performed a self-review of my own code
- [x] Followed the style guidelines of this project
- [ ] Made corresponding changes to the documentation
- [x] Didn't generate new warnings
- [x] Didn't generate failures on existing tests
- [x] Added tests that prove my fix is effective or that my feature works

## Review

When reviewing a PR, please indicate intention in comments using the following emojis:
* :cake: = Nice to have but not essential.
* :bulb: = Suggestion or a comment based on personal opinion
* :hammer: = I believe this should be changed.
* :thinking: = I don’t understand something, do you mind giving me more context?
* :rocket: = Feedback

### CODE REVIEW

File: scripts/v2-migration/actions/TokenController.unlistClaimsReward.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const { runAction } = require('./utils');
+
+async function action({ tokenController, signer, opts = {} }) {
+  const tx = await tokenController.connect(signer).unlistClaimsReward(opts);
```
Comment (danoctavian) [inline scripts/v2-migration/actions/TokenController.unlistClaimsReward.js:4]: added this pre-emptively

File: scripts/v2-migration/actions/LegacyPooledStaking.blockV1.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const { runAction } = require('./utils');
+
+async function action({ legacyPooledStaking, signer, opts = {} }) {
+  const tx = await legacyPooledStaking.connect(signer).blockV1(opts);
```
Comment (danoctavian) [inline scripts/v2-migration/actions/LegacyPooledStaking.blockV1.js:4]: this to be removed?

based on discussion seems pointless if we simply delete the code.

Comment (roxdanila) [reply scripts/v2-migration/actions/LegacyPooledStaking.blockV1.js:4]: yep, nuke it. the removal of this function is already merged

File: scripts/v2-migration/actions/LegacyPooledStaking.migrateToNewV2Pool.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+const { runAction } = require('./utils');
+
+const ARMOR_STAKER = '0x1337def1fc06783d4b03cb8c1bf3ebf7d0593fc4';
+const FOUNDATION = '0x963df0066ff8345922df88eebeb1095be4e4e12e';
+const HUGH = '0x87b2a7559d85f4653f13e6546a14189cd5455d45';
+
+async function action({ legacyPooledStaking, signer, selectedStakers = [FOUNDATION, HUGH, ARMOR_STAKER], opts = {} }) {
+  console.log('Checking that selected stakers cannot withdraw independently');
```
Comment (roxdanila) [inline scripts/v2-migration/actions/LegacyPooledStaking.migrateToNewV2Pool.js:8]: pls delete

File: scripts/v2-migration/actions/LegacyPooledStaking.migrateToNewV2Pool.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  for (const staker of selectedStakers) {
+    // pooledStaking.withdraw(uint) is not verified here for simplicity; it follows the exact same code path
```
Comment (roxdanila) [inline scripts/v2-migration/actions/LegacyPooledStaking.migrateToNewV2Pool.js:10]: delete

File: test/fork/migration-v2.js [hunk 5]
```diff
@@ -787,14 +794,18 @@ describe('V2 upgrade', function () {
   });
 
   it('Call function to block V1 staking', async function () {
-    const tx = await this.pooledStaking.blockV1();
-    await tx.wait();
+    // TODO: to be removed once blockV1 is gone
```
Comment (roxdanila) [inline test/fork/migration-v2.js:797]: nuke it

**Pull request closed without merging**